stable/osg-frontier-squid/osg-frontier-squid/values.yaml

```
@@ -54,3 +54,14 @@ SquidConf:
   # Set this to null to be randomly assigned an available node
 NodeSelection:  
   Hostname: null
+
+  #If extra configuration options are wanted, the values for said options should be inserted here
+  #By default these extra configuration options are turned off, set ExtraConfig to True to turn them on
+
+ExtraConf:
+  Enabled: false
+  acl_NET_LOCAL_src: "10.0.0.0/8 172.16.0.0/12 192.168.0.0/16"
```

```suggestion
  acl_NET_LOCAL_src: "10.0.0.0/8 172.16.0.0/12 192.168.0.0/16"
```
This is being set elsewhere